### PR TITLE
More warning removal

### PR DESCRIPTION
**Pull request by dstansby**

### CODE REVIEW

File: sunpy/net/fido_factory.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 `Fido.fetch <sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch>`.
 
 """
-from collections import Sequence
```
Comment (Cadair) [inline sunpy/net/fido_factory.py]: @nabobalis Should should backport these ones for 3.8 compatibility?

Comment (nabobalis) [reply sunpy/net/fido_factory.py]: Yes.

File: setup.cfg [hunk 2]
```diff
@@ -83,6 +83,8 @@ addopts = --doctest-rst -m "not figure"
 markers =
     online: marks this test function as needing online connectivity.
     figure: marks this test function as using hash-based Matplotlib figure verification. This mark is not meant to be directly applied, but is instead automatically applied when a test function uses the @sunpy.tests.helpers.figure_test decorator.
+    flaky
+    array_compare
```
Comment (nabobalis) [inline setup.cfg]: What do these do?

Comment (dstansby) [reply setup.cfg]: pytest raises a warning now if any markers used aren't in this list (presumably so users have a definitive list of markers?)

Comment (nabobalis) [reply setup.cfg]: Could you add some descriptions like the other two?

Comment (Cadair) [reply setup.cfg]: Both of these are really fails on the plugin to not register their marks.

**Comment by nabobalis** [comment]
Thanks @dstansby

**Pull request merged**